src/gt4py/cartesian/utils/field.py

```diff
@@ -0,0 +1,75 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import numbers
+from typing import Tuple
```

Comment:
fyi, this import is ok, as pure typing imports are excluded from the rule, see https://google.github.io/styleguide/pyguide.html#2241-exemptions